<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindDialog</class>
 <widget class="QDialog" name="BindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>82</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>绑定银行卡</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/icons/wallet.png</normaloff>:/new/icons/wallet.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	border:0.5px solid white;
	border-radius:6px;
	background-color:rgb(222,255,241);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border:1px solid white;
	border-radius:10px;
	
	
	background-color: rgb(101, 229, 255);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:20pt;
	font-weight:bold;
	color:white;
}

</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;绑定银行卡&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="bankSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(158, 232, 255);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}

</string>
       </property>
       <property name="placeholderText">
        <string>银行卡</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="accountIDIn">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(158, 232, 255);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}

</string>
       </property>
       <property name="placeholderText">
        <string>账户</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="pwIn">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(158, 232, 255);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}

</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="applyB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
